app/views/users/dashboard/index.html.erb

```diff
@@ -0,0 +1,13 @@
+
+<p>User Name: <%= @user.user_info[:username] %></p>
+<p>Number of Scores: <%= @user.user_info[:ownedPublicScoresCount] %></p>
```

Comment:
also same idea here, wondering if a user_info object would be cleaner, might not for this case though